tests/hopp/test_site_shape_tools.py

```diff
@@ -0,0 +1,147 @@
+from pytest import approx
+from shapely.geometry import Polygon
+import hopp.simulation.technologies.sites.site_shape_tools as shape_tools
+
+def test_circle_area():
+    area_m2 = 1e3
+    x0 = 0.0
+    y0 = 0.0
+    polygon, vertices = shape_tools.make_circle(area_m2, x0 = x0, y0 = y0)
+    assert polygon.area == approx(area_m2,rel = 0.1)
+
+def test_square_area():
+    area_m2 = 1e3
+    x0 = 0.0
+    y0 = 0.0
+    polygon, vertices = shape_tools.make_square(area_m2, x0 = x0, y0 = y0)
+    assert polygon.area == approx(area_m2,rel = 0.1)
+
+def test_rectangle_area():
+    area_m2 = 1e3
+    x0 = 0.0
+    y0 = 0.0
+    aspect_ratio = 1.5
+    polygon, vertices = shape_tools.make_rectangle(area_m2, aspect_ratio = aspect_ratio, x0 = x0, y0 = y0)
+    assert polygon.area == approx(area_m2,rel = 0.1)
+
+def test_hexagon_area():
+    area_m2 = 1e3
+    x0 = 0.0
+    y0 = 0.0
+    polygon, vertices = shape_tools.make_hexagon(area_m2, x0 = x0, y0 = y0)
+    assert polygon.area == approx(area_m2,rel = 0.1)
+
+def test_circle_vertices_default():
+    area_m2 = 1e3
+    x0 = 0.0
+    y0 = 0.0
+    polygon, vertices = shape_tools.make_circle(area_m2, x0 = x0, y0 = y0)
```

Comment:
I think the `x0` and `y0` shouldn't be passed here if the point is to test the defaults.